howso/prediction_stats.amlg

```diff
@@ -30,16 +30,34 @@
 	; robust_hyperparameters: flag, optional. if specified, will attempt to return stats that were computed using hyperpparameters with the
 	;						  specified robust or non-robust type.
 	; weight_feature: string, optional. if specified, will attempt to return stats that were computed using this weight_feature.
-	; condition: assoc of feature->value(s)
+	; action_condition: assoc of feature->value(s), optional. If specified, will condition the action set, which is the dataset for which the prediction stats are for. 
+	;		If both 'action_condition' and 'context_condition' are provided, then all of the action cases selected by the 'action_condition' 
+	;       will be excluded from the context set, which is the set being queried to make to make predictions for the action set, effectively holding them out.
+	;		If only 'action_condition_filter_query' is specified, then only the single predicted case will be left out.
```

Comment:
This isn't a parameter here? not sure what this is about

Reply:
yes supposed just to be action_condition